src/black/files.py

```diff
@@ -31,14 +31,19 @@
 
 
 @lru_cache()
-def find_project_root(srcs: Sequence[str]) -> Path:
+def find_project_root(srcs: Sequence[str]) -> Tuple[Path, Optional[str]]:
     """Return a directory containing .git, .hg, or pyproject.toml.
 
     That directory will be a common parent of all files and directories
     passed in `srcs`.
 
     If no directory in the tree contains a marker that would specify it's the
     project root, the root of the file system is returned.
+
+    Returns a tuple with the first element as the project root path and the
+    second element as a string which describes the method by which black
+    found out the project root (None, if `None` of the registered method
+    was used).
```

Comment:
```suggestion
    Returns a two-tuple with the first element as the project root path and
    the second element as a string describing the method by which the
    project root was discovered (it will be None the fallback root was used).
```